mrp_subcontracting_partner_management/security/res_partner_security.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_display_inventory" model="res.groups">
+        <field name="name">Display Inventory section on Vendor</field>
```

Comment:
This extra group is overkilling for such tiny feature. There are too much technical groups. You can use instead `base_technical_features` group to have technical fields without developer mode.

Reply:
Hi @pedrobaeza , our goal is to display a read-only "Subcontractor Location" field to inventory user, in order to allow user to check the subcontracting location is correct (but not change it)

A possible solution is to add a read-only field with the subcontractor location name set in the field, in order to achieve the goal above and to maintain the possibility to change the value for user with dev mode access

Reply:
It's a bit of nonsense if this module is automating all the handling of these locations, to have such field to be manually reviewed.

Reply:
Yeah, that can be an interesting added value, but without extra security group.